prompts/csharp-mstest.prompt.md

```diff
@@ -11,7 +11,7 @@ Your goal is to help me write effective unit tests with MSTest, covering both st
 ## Project Setup
 
 - Use a separate test project with naming convention `[ProjectName].Tests`
-- Reference Microsoft.NET.Test.Sdk, MSTest.TestAdapter, and MSTest.TestFramework packages
+- Reference MSTest package
```

Comment:
Consider noting `MSTest.Sdk`

Reply:
I think we can suggest it, but not all repos use MTP and mstest.sdk is still not nicely supported in VS because of the nuget tooling, so I would not default to it.